tasks/test.js

```diff
@@ -1,29 +1,32 @@
 var _ = require('lodash');
 module.exports = function (grunt) {
-  grunt.registerTask('test', function () {
+  grunt.registerTask('test:server', [ 'simplemocha:all' ]);
+  grunt.registerTask('test:browser', [ 'run:testServer', 'karma:unit' ]);
+
+  grunt.registerTask('test:quick', [
+    'test:server',
+    'test:browser'
+  ]);
+
+  grunt.registerTask('test:dev', [
+    'run:devTestServer',
+    'karma:dev'
+  ]);
+
+  grunt.registerTask('test', function (subTask) {
+    if (subTask) grunt.fail.fatal(`invalid task "test:${subTask}"`);
+
     if (grunt.option('quick')) {
-      grunt.task.run('quick-test');
+      grunt.task.run('test:quick');
       return;
     }
 
     grunt.task.run(_.compact([
       'eslint:source',
-      'simplemocha:all',
-      'maybeStartTestServer',
-      'karma:unit'
+      'test:server',
```

Comment:
maybe using `test:quick` here would make sense, so we only need to add new test types in 1 place?